hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,4 @@
+RELEASE_TYPE: minor
+
+This minor release adds functionality to :obj:`~hypothesis.settings` allowing
+it to be used as a decorator on :obj:`~hypothesis.stateful.RuleBasedStateMachine`.
```

Comment:
The Travis build is failing because - it turns out- we don't actually have a link target for `RuleBasedStateMachine`!  The best solution would be to add an `.. autoclass::` entry to [the relevant section of `docs/stateful.rst`](https://hypothesis.readthedocs.io/en/latest/stateful.html#rule-based-state-machines), and check that the class docstring makes sense in this context.

You could also mention `GenericStateMachine` in the changelog, since it works for them too!

Finally, we like to have an explicit note thanking new contributors at the end - so if you could add a final line `Thanks to Tyler Nickerson for this feature in #hacktoberfest!`